keycloak.tf

```diff
@@ -69,7 +71,7 @@ resource "aws_ecs_task_definition" "keycloak-ecs-taskdef" {
         {"name":"KC_LOG_LEVEL", "value":"INFO,cz.integsoft:DEBUG,org.infinispan:DEBUG,org.jgroups:DEBUG"},
         {"name":"KC_PROXY", "value":"edge"},
         {"name":"KC_HEALTH_ENABLED", "value":"true"},
-        {"name":"JAVA_OPTS_APPEND", "value":"-Xmx1500m -DawsRegion=${var.aws_region} -DawsJmsQueues=${var.aws_jms_queues}"},
+        {"name":"JAVA_OPTS_APPEND", "value":"-Xmx4000m -DawsRegion=${var.aws_region} -DawsJmsQueues=${var.aws_jms_queues}"},
```

Comment:
suggestion(non-blocking): might want to make this a variable as well, something like:
```
locals {
  keycloak_task_memory = 4096
  keycloak_java_memory = local.keycloak_task_memory - 500
}
...
{"name":"JAVA_OPTS_APPEND", "value":"-Xmx${local.keycloak_java_memory}m -DawsRegion=${var.aws_region} -DawsJmsQueues=${var.aws_jms_queues}"},

Reply:
That's a great suggestion! https://github.com/mbta/terraform-keycloak-sso/pull/31/commits/f56e3a9b3a087af5358bac47f6ef068abf09bf9a